src/kernel/messages.c

```diff
@@ -194,13 +194,15 @@ message *msg_message(const char *name, const char *sig, ...)
         }
         else {
             log_error("invalid parameter %s for message type %s\n", paramname, mtype->name);
+            assert(!"invalid parameter for message type");
```

Comment:
Das log gibt es auch in den Tests, ja. Man kann da inzwischen sogar einen eigenen Hook installieren, damit man log-messages testen kann (siehe log.test.c)